tailwind.config.js

```diff
@@ -18,6 +18,10 @@ module.exports = {
         secondary: '#F4DCF4',
         'app-background': '#fff',
         orange: '#ff5a42',
+        listCategory: '#f5f5dc',
+        listCategoryActive: '#81816d',
+        income: 'rgba(97,255,42,.43)',
```

Comment:
call this plannedIncome and plannedExpense because there will be separate colors for actualIncome and actualExpense

Reply:
changed It.